tulip/interfaces/gr1c.py

```diff
@@ -430,42 +430,15 @@ def check_realizable(spec, init_option="ALL_ENV_EXIST_SYS_INIT"):
         logger.info(p.stdout.read() )
         return False
 
-def synthesize(spec, init_option="ALL_ENV_EXIST_SYS_INIT"):
+def synthesize(spec):
     """Synthesize strategy realizing the given specification.
 
     @type spec: L{GRSpec}
-    @param spec: specification, which is incomplete without an
-        interpretation of initial conditions (init_option).
-
-    @type init_option: str
-    @param init_option: string declaration of the initial condition
-        interpretation to use.  This parameter corresponds to that of
-        the -n command-line flag of gr1c.  It is one of:
-
-            - "ALL_ENV_EXIST_SYS_INIT" (default) - For each initial
-              valuation of environment variables that satisfies
-              spec.env_init, the strategy must provide some valuation
-              of system variables that satisfies spec.sys_init.  Only
-              environment variables (spec.env_vars) may appear in
-              spec.env_init, and only system variables (spec.sys_vars)
-              may appear in spec.sys_init.
-
-            - "ALL_INIT" - Any state that satisfies the conjunction of
-              spec.env_init an spec.sys_init can occur initially.
-
-            - "ONE_SIDE_INIT" - At most one of spec.env_init and
-              spec.sys_init is nonempty, and the nonempty one can
-              include both environment and system variables
-              (spec.env_vars and spec.sys_vars, respectively).  Both
-              being empty is equivalent to spec.env_init=["True"].  If
-              spec.env_init is nonempty, then any state satisfying it
-              is possible initially.  If spec.sys_init is nonempty,
-              then the strategy need only choose one initial state
-              satisfying it.
-
-        Consult the U{documentation of gr1c
-        <https://tulip-control.github.io/gr1c/md_spc_format.html#initconditions>}
-        for detailed descriptions.
+    @param spec: specification.
+
+    Consult the U{documentation of gr1c
```

Comment:
Because `select_options()` is called within `synthesize()`, and because their relationship is not apparent from their docstrings, I agree about leaving the URL in the docstring of `synthesize()`.